kbchat/kvstore_test.go

```diff
@@ -0,0 +1,60 @@
+package kbchat
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// test doesn't make assumption about prior state
+func TestKVStore(t *testing.T) {
+	alice, dir := testBotSetup(t, "alice")
+	channel := getTeamChatChannel(t, "acme")
+	team := channel.Name
+	defer testBotTeardown(t, alice, dir)
+
+	namespace := "_test_namespace1"
+	key := "_test_key1"
+
+	// put with default revision
+	res2, err2 := alice.PutEntry(team, namespace, key, "value1", 0)
+	require.NoError(t, err2)
+	rev := res2.Revision
+
+	// fail put
+	_, err3 := alice.PutEntry(team, namespace, key, "value2", rev)
+	require.Error(t, err3)
+
+	// list namespaces
+	res4, err4 := alice.ListNamespaces(team)
+	require.NoError(t, err4)
+	require.True(t, len(res4.Namespaces) > 0)
```

Comment:
suggest we also verify that we have the namespace we expect. this will verify that the types and json parsing are correct.